### ISSUE TITLE
Templates are not installed from setup.py

### ISSUE THREAD

**Issue by lost-theory** [playpauseandstop/Flask-Dropbox#3]
I think you may need to add the `package_data` option to your setup.py to fix.

Take a look at e.g. flask-peewee:

https://github.com/coleifer/flask-peewee/blob/master/setup.py#L17

**Comment by playpauseandstop** [comment]
Strange, I've added

```
include flask_dropbox/templates/dropbox/*.html
```

line to `MANIFEST.in` and `python setup.py sdist` already included that file to source distribution:

```
arpegius:Flask-Dropbox playpauseandstop$ python setup.py sdist
/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/distutils/dist.py:267: UserWarning: Unknown distribution option: 'install_requires'
  warnings.warn(msg)
running sdist
running check
warning: sdist: standard file not found: should have one of README, README.txt

reading manifest template 'MANIFEST.in'
writing manifest file 'MANIFEST'
creating Flask-Dropbox-0.1.2
creating Flask-Dropbox-0.1.2/flask_dropbox
creating Flask-Dropbox-0.1.2/flask_dropbox/templates
creating Flask-Dropbox-0.1.2/flask_dropbox/templates/dropbox
making hard links in Flask-Dropbox-0.1.2...
hard linking LICENSE -> Flask-Dropbox-0.1.2
hard linking MANIFEST.in -> Flask-Dropbox-0.1.2
hard linking README.rst -> Flask-Dropbox-0.1.2
hard linking setup.py -> Flask-Dropbox-0.1.2
hard linking flask_dropbox/__init__.py -> Flask-Dropbox-0.1.2/flask_dropbox
hard linking flask_dropbox/blueprint.py -> Flask-Dropbox-0.1.2/flask_dropbox
hard linking flask_dropbox/settings.py -> Flask-Dropbox-0.1.2/flask_dropbox
hard linking flask_dropbox/utils.py -> Flask-Dropbox-0.1.2/flask_dropbox
hard linking flask_dropbox/views.py -> Flask-Dropbox-0.1.2/flask_dropbox
hard linking flask_dropbox/templates/dropbox/callback.html -> Flask-Dropbox-0.1.2/flask_dropbox/templates/dropbox
Creating tar archive
removing 'Flask-Dropbox-0.1.2' (and everything under it)
```

So, which version of `Flask-Dropbox` did you use?

**Comment by lost-theory** [comment]
I used the version that was on pypi (Flask-Dropbox-0.1.1.tar.gz). I think that MANIFEST.in handles putting the file into the distribution (the tgz), but `package_data` in setup.py is what handles actually installing the file. For example, with the new 0.1.2 you just released:

https://gist.github.com/3091483

**Comment by playpauseandstop** [comment]
ok, got you, thanks for reporting

will fix this in some minutes

**Comment by playpauseandstop** [comment]
Fixed in 410d1588df449da854373a8e7925defa2c1c0fa4. Closed
